### PR TITLE
July172020

### PR DESCRIPTION
**Pull request by alpisin**
test

### CODE REVIEW

**Comment by alpisin** [comment]
Please Review

File: Scripts/ITG DATA PULL.c [hunk 1]
```diff
@@ -2,5 +2,12 @@
 int main() {
  
    printf("ITG DATA PULL");
+   printf("ITG DATA PULL v2");
+   printf("ITG DATA PULL v3");
+   printf("ITG DATA PULL v4");
+   printf("ITG DATA PULL v5");
+   
    return 0;
+   //test
```
Comment (CemDoganay) [inline Scripts/ITG DATA PULL.c:11]: I am not sure about this

File: Scripts/ITG DATA PULL.c [hunk 2]
```diff
@@ -2,5 +2,12 @@
 int main() {
  
    printf("ITG DATA PULL");
+   printf("ITG DATA PULL v2");
+   printf("ITG DATA PULL v3");
+   printf("ITG DATA PULL v4");
+   printf("ITG DATA PULL v5");
```
Comment (CemDoganay) [inline Scripts/ITG DATA PULL.c:8]: might create an issue

File: Scripts/ITG DATA PULL.c [hunk 3]
```diff
@@ -2,5 +2,12 @@
 int main() {
  
    printf("ITG DATA PULL");
+   printf("ITG DATA PULL v2");
```
Comment (CemDoganay) [inline Scripts/ITG DATA PULL.c:5]: is it better

**Pull request closed without merging**